src/app/freelancer/market/page.tsx

```diff
@@ -209,6 +209,17 @@ const Market: React.FC = () => {
           </div>
           <DropdownProfile />
         </header>
+        <div className="mb-8 ml-10">
+          <h1 className="text-3xl font-bold">
+            {' '}
```

Comment:
don't do this to add space
use css margin or padding

Reply:
ok